app/views/spree/admin/shipping_categories/new.html.haml

```diff
@@ -5,7 +5,7 @@
 
 - content_for :page_actions do
   %li
-    = button_link_to Spree.t(:back_to_shipping_categories_list), spree.admin_shipping_categories_path, :icon => 'icon-arrow-left'
+    = button_link_to Spree.t(:back_to_shipping_categories), spree.admin_shipping_categories_path, :icon => 'icon-arrow-left'
```

Comment:
Test had been failing due to a `missing translation key` so I fixed it here

Reply:
Great. If you had done this in a separate commit then you could have added the explanation there. Everyone looking through the codes history can then see this context.

Reply:
Thanks for the tip @mkllnk, I should use my commit messages to express myself more 👍